### ISSUE TITLE
bash redirection error

### ISSUE THREAD

**Issue by GentleInfant** [google-deepmind/alphafold#896]
When running `docker build -f docker/Dockerfile -t alphafold .` using `main` on an Ubuntu install with docker emulation provided by podman I'm getting the following output (the images have already been downloaded, and I selected the `docker.io` repo when asked)

```
$ docker build -f docker/Dockerfile -t alphafold . 
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
STEP 1/16: FROM nvidia/cuda:11.1.1-cudnn8-runtime-ubuntu18.04
STEP 2/16: ARG CUDA
--> Using cache b6cfd975088d07362916fb77eacb6922706990ed979c2c553c5fefec9487fc19
--> b6cfd975088
STEP 3/16: SHELL ["/bin/bash", "-o", "pipefail", "-c"]
--> Using cache b7a9566fcecc44cbf4b5430c29ea85935063b127c95cb2b662b2cd79d30c8052
--> b7a9566fcec
STEP 4/16: RUN apt-get update     && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y         build-essential         cmake         cuda-command-line-tools-$(cut -f1,2 -d- <<< ${CUDA//./-})         git         hmmer         kalign         tzdata         wget     && rm -rf /var/lib/apt/lists/*     && apt-get autoremove -y     && apt-get clean
/bin/sh: 1: Syntax error: redirection unexpected
Error: error building at STEP "RUN apt-get update     && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y         build-essential         cmake         cuda-command-line-tools-$(cut -f1,2 -d- <<< ${CUDA//./-})         git         hmmer         kalign         tzdata         wget     && rm -rf /var/lib/apt/lists/*     && apt-get autoremove -y     && apt-get clean": error while running runtime: exit status 2
```

From some googling it seems as though when being run as root (which I believe the script is doing) that dash rather than bash is used which doesn't support `<<<` redirection. If I replace `$(cut -f1,2 -d- <<< ${CUDA//./-})` with `$(echo $CUDA | sed 's/\./-/g' | cut -f1,2 -d-)` then it works but fails later with `/bin/sh: 1: pushd: not found` which I believe is another bash-specific command.

So tl;dr is that because of this I can't build the docker image.

**Comment by GentleInfant** [comment]
Actually closing this. Using the official docker install seems to work.
